stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channel/ChannelsScreen.kt

```diff
@@ -96,41 +97,45 @@ public fun ChannelsScreen(
     var searchQuery by rememberSaveable { mutableStateOf("") }
 
     Box(modifier = Modifier.fillMaxSize()) {
-        Column(Modifier.fillMaxSize()) {
-
-            if (isShowingHeader) {
-                ChannelListHeader(
-                    onHeaderActionClick = onHeaderClickAction,
-                    currentUser = user,
-                    title = title,
-                    connectionState = connectionState
-                )
+        Scaffold(
+            modifier = Modifier.fillMaxSize(),
+            topBar = {
```

Comment:
maybe pass in null as a topBar in case we shouldn't show it?

Reply:
The `topBar` parameter is not nullable:

```
topBar: @Composable () -> Unit = {},
```